.github/workflows/periodic.yml

```diff
@@ -1,13 +1,14 @@
 ---
-name: Markdown Link Checks
+name: Periodic
+
 on:
   schedule:
     - cron: "0 0 * * 0"
 
 jobs:
   markdown-link-check-periodic:
+    name: Markdown Links (all files)
```

Comment:
I clarified the difference between the PR-triggered/modified-files-only and periodic/all-files jobs, per discussions with @mangelajo and @mkolesnik on Slack.